_data/projects/ReactWeatherApp.yml

```
@@ -0,0 +1,14 @@
+name: React Weather Application
+desc: This is a very simple React Js based weather application, that uses Weather API. 
+site: https://stellar-pie-6964b6.netlify.app/
+tags:
+  - reactjs
+  - javascript
+  - hacktoberfest
+  - open-source
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/zetabug/react-weather-application/labels/up%20for%20grabs
```

This is the only section that will be detected on the site, but you can set the `name` and `link` to whichever label you wish to use. If you'd prefer to use `good first issue`:

```suggestion
  name: good first issue
  link: https://github.com/zetabug/react-weather-application/labels/good%20first%20issue
```